ndk/src/bitmap.rs

```diff
@@ -136,9 +136,23 @@ impl AndroidBitmapInfo {
         self.inner.stride
     }
 
-    pub fn format(&self) -> BitmapFormat {
+    /// Attempt to convert the internal, native `AndroidBitmapInfo` into
+    /// this crate's [`BitmapFormat`] type. This may fail if the value
+    /// does not have a corresponding Rust enum variant.
+    pub fn try_format(&self) -> Result<BitmapFormat, TryFromPrimitiveError<BitmapFormat>> {
         let format = self.inner.format as u32;
-        format.try_into().unwrap()
+        format.try_into()
+    }
+
+    /// Attempt to convert the internal, native `AndroidBitmapInfo` into
+    /// this crate's [`BitmapFormat`] type.
+    ///
+    /// # Panics
+    ///
+    /// This function panics if the underlying value does not have a
+    /// corresponding variant in [`BitmapFormat`].
```

Comment:
Add something like: ``Use [`try_format()`][AndroidBitmapInfo::try_format() for an infallible version of this function``.